### PR TITLE
GCC2023 updates

### PR DESCRIPTION
**Pull request by neoformit**
Some styling and content updates on GCC2023 pages from the Galaxy AU team.

- Remove toc from pages as unnecessary
- Refactor bulky header content to index page only 
- Remove hard-coded bold text from header menu
- Update header image with AU GCC2023 banner

### CODE REVIEW

**Comment by neoformit** [comment]
@afgane

File: content/events/gcc2023/index.md [hunk 1]
```diff
@@ -19,6 +19,28 @@ subsites: [all]
...
+### Brisbane, Queensland, Australia
+
+#### [#UseGalaxy2023](https://twitter.com/hashtag/UseGalaxy2023)
+
+<a target="_blank" href="https://gcc-2023.eventbrite.com.au/" type="button" class="btn btn-primary">
+    <strong>Register now</strong>
+</a>
+<br>
+<a target="_blank" href="/events/gcc2023/sponsors/" type="button" class="btn btn-primary">
+    <strong>View sponsorship opportunities</strong>
```
Comment (afgane) [inline content/events/gcc2023/index.md:39]:
```suggestion
<a target="_blank" href="https://forms.gle/ETCPW8Vnprbij7zn6" type="button" class="btn btn-primary">
    <strong>Submit an abstract</strong>
```

File: content/events/gcc2023/index.md [hunk 2]
```diff
@@ -19,6 +19,28 @@ subsites: [all]
 
 <slot name="/events/gcc2023/header" />
 
+<div class="text-center my-5">
+
+# **2023 Galaxy Community Conference (GCC2023)**
+
+### SAVE THE DATE
```
Comment (afgane) [inline content/events/gcc2023/index.md:26]:
```suggestion
```

File: content/events/gcc2023/register/index.md [hunk 3]
```diff
@@ -1,9 +1,23 @@
...
+
 <slot name="/events/gcc2023/header" />
 
+<div class="text-center my-5">
+    <a target="_blank" href="https://gcc-2023.eventbrite.com.au/" type="button" class="btn btn-primary center">
+        <strong>Register now</strong>
+    </a>
+    <br>
+    <a target="_blank" href="/events/gcc2023/sponsors/" type="button" class="btn btn-primary center">
+        <strong>View sponsorship opportunities</strong>
```
Comment (afgane) [inline content/events/gcc2023/register/index.md:13]:
```suggestion
    <a target="_blank" href="https://forms.gle/ETCPW8Vnprbij7zn6" type="button" class="btn btn-primary center">
        <strong>Submit an abstract</strong>
```

File: content/events/gcc2023/header.md [hunk 4]
```diff
@@ -1,47 +1,21 @@
...
-<!-- https://commons.wikimedia.org/wiki/File:The_heart_reef,_part_of_the_Great_Barrier_Reef_near_Airlie_Beach,_Whitsunday_Islands,_Queensland.jpg -->
-[![gcc2023](/images/events/gcc2023/au-heart-island.png)](/events/gcc2023/)
+[![gcc2023](/images/events/gcc2023/au-gcc-banner.png)](/events/gcc2023/)
 
 </div>
 <div class="linkbox-horizontal trim-p">
 
 [Home](/events/gcc2023/) |
-[**Abstracts**](/events/gcc2023/abstracts/) |
+[Abstracts](/events/gcc2023/abstracts/) |
```
Comment (afgane) [inline content/events/gcc2023/header.md:10]: Given how many links are here, I like highlighting 1 or 2 as a way to draw users's attention to the links that are most time-pertinent... But that's just my opinion.

**Review by afgane — APPROVED** [review]

File: content/events/gcc2023/header.md (hunk 4 above)
Comment (neoformit) [reply content/events/gcc2023/header.md:10]: It is a bit weird/confusing though, because that usually infers which page is currently active!

**Comment by neoformit** [comment]
@bgruening

**Comment by bgruening** [comment]
Thanks.

**Pull request merged**